entry.sh

```diff
@@ -56,6 +56,10 @@ fi
 # not a problem.
 modprobe ip6_tables || true
 
+# Cleanup leftover Supervisor-created lockfiles from any previous processes.
+# Supervisor-created lockfiles have a UID of 65534.
+find /mnt/root/tmp/balena-supervisor/services -type f -user 65534 -name "*updates.lock" | xargs rm
```

Comment:
I think is better if we add something like

```suggestion
export BASE_LOCK_DIR="/tmp/balena-supervisor/services"
export LOCKFILE_UID=65534
find "/mnt/root/${BASE_LOCK_DIR}" -type f -user "${LOCKFILE_UID}" -name "*updates.lock" | xargs rm
```

And then use these exported variables in the update-lock module

Reply:
Done, will be in the next force-push